src/modules/fundingManager/bondingCurve/FM_BC_Bancor_Redeeming_VirtualSupply_v1.sol

```diff
@@ -592,4 +592,29 @@ contract FM_BC_Bancor_Redeeming_VirtualSupply_v1 is
             );
         }
     }
+
+    //--------------------------------------------------------------------------
+    // Overriden Internal Functions
+
+    /// @dev    Distributes issuance tokens by minting them to the receiver.
+    /// @param  _receiver The address that will receive the bought tokens.
```

Comment:
> [...] receive the **bought** tokens.

Is this really just for buying tokens? Didn't check, if so then that's okay probably, but I believe we'll likely also use this to distribute fees or whatnot.

Reply:
See comment above